ci-operator/step-registry/interop-tests/cnv-tests-smoke/interop-tests-cnv-tests-smoke-commands.sh

```diff
@@ -4,165 +4,41 @@ set -o nounset
-      image: busybox
-      command:
-        - "sh"
-        - "-c"
-        - "while true ; do echo ready ; sleep 30 ; done"
-      volumeMounts:
-        - name: cnv-tests-results
-          mountPath: /data/results
-  restartPolicy: Never
-  initContainers:
-    - name: download-binaries
-      envFrom:
-        - configMapRef:
-            name: cluster-info
-      image: quay.io/openshift-cnv/cnv-tests:${CNV_TESTS_IMAGE_TAG}
-      command:
-        - /bin/sh
-        - -c
-        - |
-            set -ex
-            cd /binaries
-
-            curl -ksSL https://downloads-openshift-console.apps.${CLUSTER_NAME}.${BASE_DOMAIN}/amd64/linux/oc.tar \
-              | tar -xvf -
-            ln -sf oc kubectl
-
-            curl -ksSL $(oc get consoleclidownload virtctl-clidownloads-kubevirt-hyperconverged \
-                --output=jsonpath='{range $.spec.links[*]}{$.href}{"\n"}{end}' \
-                | grep -Fw linux
-            ) | tar -xzvf -
-
-      volumeMounts:
-        - name: cnv-tests-binaries
-          mountPath: "/binaries"
-  securityContext:
-    privileged: true
-  volumes:
-    - name: cnv-tests-ssh-key
-      configMap:
-        name: cnv-tests-ssh-key
-    - name: cnv-tests-binaries
-      emptyDir: {}  # Populated in initContainers
-    - name: cnv-tests-cluster-auth
-      configMap:
-        name: cnv-tests-cluster-auth
-    - name: cnv-tests-results
-      emptyDir: {}
-EOF
-
-oc wait \
-    --for=condition='Ready' \
-    --timeout='10m' \
-    -n ${CNV_TESTS_NAMESPACE} pod cnv-tests
-
-# Stream test logs into the console
-oc logs -n ${CNV_TESTS_NAMESPACE} -f cnv-tests -c test-runner --tail 50
-while true
-do
-    if oc exec -n ${CNV_TESTS_NAMESPACE} cnv-tests -- pgrep -x "pytest" &> /dev/null;
-    then
-        oc logs -n ${CNV_TESTS_NAMESPACE} -f cnv-tests -c test-runner --tail 50
-    else
-        break
-    fi
-done
+COLLECTOR_CONF_FILE="${ARTIFACT_DIR}/containerized-data-collector.yaml"
+
+cat << __EOF__ | tee "${COLLECTOR_CONF_FILE}"
+data_collector_base_directory: "/${ARTIFACT_DIR}/tests-collected-info"
+collect_data_function: "ocp_wrapper_data_collector.data_collector.collect_data"
+collect_pod_logs: true
+__EOF__
+
+set -x
+START_TIME=$(date "+%s")
+
+poetry run pytest tests \
+  --pytest-log-file="${ARTIFACT_DIR}/pytest.log" \
+  --data-collector="${COLLECTOR_CONF_FILE}" \
+  --junit-xml="${ARTIFACT_DIR}/xunit_results.xml" \
+  --tc-file=tests/global_config.py \
+  --tc-format=python \
+  --tc default_storage_class:ocs-storagecluster-ceph-rbd \
+  --tc default_volume_mode:Block \
+  --latest-rhel \
+  --tb=native \
+  --storage-class-matrix=ocs-storagecluster-ceph-rbd \
+  -o log_cli=true \
+  -m smoke || /bin/true
+
+FINISH_TIME=$(date "+%s")
+DIFF_TIME=$((FINISH_TIME-START_TIME))
+set +x
+
+if [[ ${DIFF_TIME} -le 600 ]]; then
+    echo ""
+    echo " 🚨  The tests finished too quickly (took only: ${DIFF_TIME} sec), pausing here to give us time to debug"
+    echo "  😴 😴 😴"
+    sleep 7200
```

Comment:
Looks like this line is used for debug ?

Reply:
Yes, this should give us 2h to do some investigation, without the need to reprovision the cluster again, just to verify it and catch it just at the right time.

Reply:
@liangxia Can we resolve this thread and merge the PR?

Reply:
Ack